switchboard/conditions.py

```diff
@@ -36,44 +52,54 @@ def set_values(self, name):
         if name and not self.label:
             self.label = titlize(name)
 
-    def is_active(self, condition, value):
-        return condition == value
+    def is_active(self, value, actual_value):
+        return value == actual_value
 
     def validate(self, data):
         value = data.get(self.name)
         if value:
             value = self.clean(value)
-            assert isinstance(value, basestring), 'clean methods must return strings'
+            is_string = isinstance(value, basestring)
+            assert is_string, 'clean methods must return strings'
         return value
 
-    def clean(self, value):
+    def clean(self, value):  # pragma: nocover
```

Comment:
I think there's an interesting philosophical debate we could get into, but not worth having for this project right now :)  I'm ok with your approach.